### ISSUE TITLE
release a new version

### ISSUE THREAD

**Issue by MoLow** [nodejs/tap2junit#41]
Blocked by https://github.com/nodejs/tap2junit/issues/41#issuecomment-1466024702

there are some fixes waiting for deployment

**Comment by aduh95** [comment]
Who can release new versions for this package? https://pypi.org/project/tap2junit/ lists @cclauss and @jbergstroem, I don't know if it's still up to date.

**Comment by cclauss** [comment]
I can.  I have been waiting for Py3.11 but now that it is out, I need to get cracking...

**Comment by cclauss** [comment]
@MoLow Can you please create a new pull request that follows the pattern of
* #12

We will use the approval of PR as the trigger for releasing to PyPI.  https://pypi.org/project/tap2junit
